### PR TITLE
CNV-32105: Add VM actions icons to the VM details page

### PR DESCRIPTION
**Pull request by pcbailey**
## 📝 Description

This PR adds VM action icons to the title bar of the VM details page allowing for easy control of the basic operations of the VM. It also removes the VM actions from the Details card kebab menu.

Jira issue: https://issues.redhat.com/browse/CNV-32105

## 🎥 Demo

https://github.com/kubevirt-ui/kubevirt-plugin/assets/8544299/c3cd9a63-aef0-4d20-b2bb-448d24d7fd96

### CODE REVIEW

**Comment by apinnick** [comment]
Will this be released with 4.14?

**Comment by pcbailey** [comment]
> Will this be released with 4.14?

@apinnick No, this is a 4.15 feature.

File: src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx [hunk 1]
```diff
@@ -70,6 +71,7 @@ const VirtualMachineNavPageTitle: FC<VirtualMachineNavPageTitleProps> = ({ name,
               <SidebarEditorSwitch />
             </SplitItem>
           )}
+          {!isEmpty(vm) && <VMActionsIconBar vm={vm} />}
```
Comment (metalice) [inline src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx:74]: u can move it 2 lines below and add <> </> and remove the double check

File: src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/VirtualMachinesOverviewTabDetails.tsx [hunk 2]
```diff
@@ -81,7 +83,9 @@ const VirtualMachinesOverviewTabDetails: FC<VirtualMachinesOverviewTabDetailsPro
   return (
     <div className="VirtualMachinesOverviewTabDetails--details">
       <Card>
-        <VirtualMachinesOverviewTabDetailsTitle vm={vm} />
```
Comment (metalice) [inline src/views/virtualmachines/details/tabs/overview/components/VirtualMachinesOverviewTabDetails/VirtualMachinesOverviewTabDetails.tsx:84]: why removing? this component in any other use or can be deleted?

File: src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx (hunk 1 above)
Comment (pcbailey) [reply src/views/virtualmachines/details/VirtualMachineNavPageTitle.tsx]: Done!

**Pull request merged**